ROADMAP.md

```diff
@@ -0,0 +1,51 @@
+# Minikube Roadmap
+This document contains the goals, plans, and priorities for the minikube project.
+
+## Goals
+The primary goal of minikube is to make it simple to run Kubernetes on your laptop, both for getting started and day-to-day development workflows.
+Here are some specific features that align with our goal:
+* Single command setup and teardown UX.
+* Unified UX across OSes
+* Minimal dependencies on third party software.
+* Minimal resource overhead.
+* Replace any other alternatives to local cluster deployment.
+
+## Non-Goals
+* Simplifying kubernetes production deployment experience. Kube-deploy is attempting to tackle this problem.
+* Supporting all possible deployment configurations of Kubernetes like various types of storage, networking, etc.
+
+## Priorities
+This section contains the overall priorities of the minikube project, in rough order.
+
+ * Setting up a well-tested, secure and complete Kubernetes cluster locally.
+ * Keeping up with new Kubernetes releases and features.
+   * Load Balancer support
+   * Persistent disks
+ * Mac OSX and Linux support initially, Windows support later.
```

Comment:
Since we are talking about priorities here, can you place `Windows Support` somewhere appropriate in this list?